packages/admin/cms-admin/src/dam/DataGrid/FolderDataGrid.tsx

```diff
@@ -215,13 +215,14 @@ const FolderDataGrid = ({
                 },
             });
 
-            const position = result.data.damItemListPosition;
-            const targetPage = Math.floor(position / dataGridProps.paginationModel.pageSize);
+            //const position = result.data.damItemListPosition;
+            //const targetPage = Math.floor(position / dataGridProps.paginationModel.pageSize);
 
             if (redirectToSubfolder && id !== redirectedToId && parentId && parentId !== currentFolderId) {
                 switchApi.activatePage("folder", parentId);
             } else {
-                dataGridProps.onPaginationModelChange?.({ page: targetPage, pageSize: dataGridProps.paginationModel.pageSize }, {});
+                //TODO: find a suitable way to pass api here
+                //dataGridProps.onPaginationModelChange?.({ page: targetPage, pageSize: dataGridProps.paginationModel.pageSize }, {});
```

Comment:
Unsure how to fix this right now, because the second argument of onPagionationModelChange needs an apiRef of the datagrid,...

Reply:
fixed it with useGridApiRef